pkg/api/api.go

```diff
@@ -52,7 +52,7 @@ type ConfigHandler func(actualCfg interface{}, defaultCfg interface{}) http.Hand
 type Config struct {
 	SkipLabelNameValidationHeader bool `yaml:"skip_label_name_validation_header_enabled" category:"advanced"`
 
-	EnableOtelMetadataStorage bool `yaml:"enable_otel_metadata_translation" category:"experimental"`
+	EnableOtelMetadataStorage bool `yaml:"enable_otel_metadata_translation" category:"deprecated"`
```

Comment:
Can we please add comment to remove this option in Mimir 2.14?

Reply:
Yep, can do.

Reply:
Done.